test/testtokenize.cpp

```diff
@@ -7026,6 +7027,9 @@ class TestTokenizer : public TestFixture {
         ASSERT_EQUALS("Cc& MakeSpancdata.(csize.(,(",
                       testAst("template <typename C> constexpr auto MakeSpan(C &c) -> decltype(MakeSpan(c.data(), c.size())) {}"));
     }
+    void astnoexcept() {
+        ASSERT_EQUALS("noexceptaswap.b((", testAst("void f() noexcept(noexcept(a.swap(b))) {}"));
```

Comment:
can a noexcept contain a noexcept? shouldn't it be just `void f() noexcept(a.swap(b)) {}`?